en/flight_modes_rover/ackermann.md

```diff
@@ -14,30 +14,92 @@ Selecting any other mode than those listed below will either stop the rover or c
 
 Manual modes require stick inputs from the user to drive the vehicle.
 
-![Manual Controls](../../assets/airframes/rover/flight_modes/rover_manual_controls.png)
+![Manual Controls](../../assets/airframes/rover/flight_modes/manual_controls_ackermann_rover.png)
```

Comment:
FWIW the left right being labelled lateral accel. "feels" odd. I don't know why, maybe because it feels like turning left/right :-)

Reply:
I might be missunderstanding your comment, but the lateral acceleration is for turning left/right.
Ackermann (unlike the other rover modules) has different "meanings" for the stick inputs based on the mode:
- Manual: Directly control steering angle which leads to cornering.
- All other manual modes: Control lateral acceleration which is then mapped to a steering angle as follows:

$$ \theta = \arctan(\frac{w_b \cdot a_{lat}}{ v^2}) $$

with

- $w_b:$ Wheel base,
- $\theta:$ Steering angle, anbd
- $v:$ Forward speed.

Also if we look at the lateral acceleration as a vector, it is poining in the direction as indicated in the picture.

Reply:
What I mean is that someone seeing that diagram might think that the control literally moves you left/right as per the mechanum rover. But you still curve left/right, so the actual movement is more like yaw on plane - ie. a curved path forward and to right/left.

I never thought of this as a diagram showing how setpoints are applied.

But anyhow, the label does say acceleration so if you are happy with this, so am I!

Reply:
Thanks for the clarification, now I see what you mean. I agree with you, at this point in the document it would make more sense to just indicate the resulting movement rather then going into detail on the setpoints. I will write this down and make it part of the next docs PR.